src/bin/cql-stress-cassandra-stress/java_generate/values/mod.rs

```diff
@@ -96,6 +96,11 @@ pub trait ValueGenerator: Send + Sync + 'static {
     ) -> CqlValue;
 }
 
+#[cfg(feature = "user-profile")]
+pub trait ValueGeneratorFactory: Send + Sync {
+    fn create(&self) -> Box<dyn ValueGenerator>;
+}
+
```

Comment:
What's the purpose of this trait?
Please add docstrings.

Reply:
Also, the commit message misses context (probably knowledge about c-s internals) that is likely necessary here.